extensions/wikia/AnalyticsEngine/js/universal_analytics.js

```diff
@@ -121,66 +121,6 @@
 		);
 	}
 
-	if (window.wgIsGASpecialWiki) {
-		// Special Wikis account - UA-32132943-1/UA-32132943-2
-		if (isProductionEnv) {
-			// Production Environment
-			window.ga(
-				'create', 'UA-32132943-1', 'auto',
-				{
-					'name': 'special',
-					'sampleRate': 100,
-					'allowLinker': true,
-					'userId': window.wgGAUserIdHash
-				}
-			);
-		} else {
-			// Development Environment
-			window.ga(
-				'create', 'UA-32132943-2', 'auto',
-				{
-					'name': 'special',
-					'sampleRate': 100,
-					'allowLinker': true,
-					'userId': window.wgGAUserIdHash
-				}
-			);
-		}
-
-		// Enable Demographics and Interests Reports
-		window.ga('special.require', 'displayfeatures');
```

Comment:
should we enable that on general account?

Reply:
@hakubo makes sense to me (since we have it enabled already for ads and ve accounts), thanks for noticing!

Reply:
actually it is enabled via Admin UI, so I won't change it here.